stuff/pipelines/client_contest.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/python3
+
+'''Processing contest messages.'''
+
+import argparse
+import logging
+import os
+import sys
+import json
+import MySQLdb
+import MySQLdb.cursors
+import pika
+from verification_code import generate_code
+
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "."))
+import lib.db   # pylint: disable=wrong-import-position
+import lib.logs  # pylint: disable=wrong-import-position
+
+
+def is_contest_finish(
+        cur: MySQLdb.cursors.BaseCursor,
+        contest_id: str) -> bool:
+    '''verificate if contest is finish'''
+    cur.execute('''
+                SELECT
+                    COUNT(*) AS `count`
+                FROM
+                    `Contests`
+                WHERE
+                    `contest_id` = %s AND
+                    `finish_time` >= NOW();
+                ''', (contest_id))
+    for row in cur:
+        if row['count'] > 0:
```

Comment:
```suggestion
    (count,) = cur.fetchone()
```

si nada más hay un renglón, se prefiere no hacer el ciclo.